pkg/network/ovn_kubernetes_test.go

```diff
@@ -669,11 +677,37 @@ func TestOVNKubernetesIsSafe(t *testing.T) {
 
 	next.Migration.MTU.Network.From = prev.DefaultNetwork.OVNKubernetesConfig.MTU
 
+	// invalid Migration.MTU.Host.To, lower than minimum MTU for IPv4
+	next.Migration.MTU.Network.To = ptrToUint32(100)
+	errs = isOVNKubernetesChangeSafe(prev, next)
+	g.Expect(errs).To(HaveLen(1))
+	g.Expect(errs[0]).To(MatchError(fmt.Sprintf("invalid Migration.MTU.Network.To(%d), has to be in range: %d-%d", *next.Migration.MTU.Network.To, MinMTUIPv4, MaxMTU)))
+
+	// invalid Migration.MTU.Host.To, higher than maximum MTU for IPv4
```

Comment:
Shouldn't the comment be: `// invalid Migration.MTU.Network.To, higher than maximum MTU for IPv4`